dataprep/eda/create_report/formatter.py

```diff
@@ -115,102 +117,115 @@ def format_basic(df: dd.DataFrame, cfg: Config) -> Dict[str, Any]:
 
     # results dictionary
     res: Dict[str, Any] = {}
-
+    # figure list
+    figs: List[Figure] = []
     # overview
-    data["ov"].pop("ks_tests")
-    res["overview"] = format_ov_stats(data["ov"])
+    if cfg.overview.enable:
```

Comment:
Why do we don't have config for create report before? Is it originally planned to be done this time?